types/sanctuary/index.d.ts

```diff
@@ -152,6 +157,10 @@ declare namespace Sanctuary {
       <A, B>   (f: Fn <A, ChainRec<Either<A, B>>>): (x: A) => ChainRec<B>;
     };
     extend<A, B>(f: Fn<Extend<A>, B>): (extend_: Extend<A>) => Extend<B>;
+    duplicate<A>(comonad: A[]): A[][];
```

Comment:
Could we use `ReadonlyArray` here?

Reply:
sure, it works!
![image](https://user-images.githubusercontent.com/4733061/86545452-782a4800-bf05-11ea-85a4-75eabf8a2e68.png)